### ISSUE TITLE
[Bug] Footer accessibility issues

### ISSUE THREAD

**Issue by jaredh-um** [unimelb/pattern-lib#1552]
### Currently

- Links in the footer are coloured white and are not underlined. Because the heading/body font colour is also white, there is no way of distinguishing links from body/heading text apart from context cues like icons and placement.

- The placeholder text in the search box does not meet >4.5:1 contrast requirements

- The search input field does not have a border or background that contrasts against the container background at >3:1

### Proposed changes

- [x] **all** links in footer should be underlined by default, with underline removed on hover
- [x] adjust border properties to be on all sides with 2px radius
- [x] adjust search input field placeholder text value to be white, 60% opacity
- [x] add `margin-top` to input field of `0.25rem` to push label up
- [x] adjust padding on input so magnifying glass icon has more padding on left, adjust top value so vertically centred
- [x] adjust `padding-bottom` on input field to `.5rem` to match top padding
- [x] input border on focus should change to white for all sides

### What this should look like (example)

![Screen Shot 2021-03-09 at 1.26.52 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/38f6064d-c85a-40aa-88c9-5e73fe15cdab)
